internal/pkg/term/selector/selector.go

```diff
@@ -152,15 +154,20 @@ func NewConfigSelect(prompt Prompter, store ConfigLister) *ConfigSelect {
 
 // NewWorkspaceSelect returns a new selector that chooses applications and environments from the config store, but
 // services from the local workspace.
-func NewWorkspaceSelect(prompt Prompter, store AppEnvLister, ws WsWorkloadDockerfileLister) *WorkspaceSelect {
+func NewWorkspaceSelect(prompt Prompter, store ConfigLister, ws WorkspaceRetriever) *WorkspaceSelect {
+	summary, err := ws.Summary()
+	if err != nil {
+		return nil
+	}
```

Comment:
we should probably return `nil, fmt.Errorf("read workspace summary: %v", err)` here instead otherwise in case an error occurred here, afterwards the code will panic

Reply:
fixed